src/index.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable no-restricted-globals */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-use-before-define */
+import './styles/main.scss';
+
+const clear = document.querySelector('.clear-btn');
+const list = document.querySelector('.todo-list');
+const input = document.getElementById('add-input');
+const form = document.getElementById('todoform');
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+const red = 'linethrough';
+
+// first render
+renderTodos();
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  saveTodo();
+  localStorage.setItem('todos', JSON.stringify(todos));
+});
+
+function saveTodo() {
+  const todoValue = input.value;
+
+  todos.push({
+    value: todoValue,
+    completed: false,
+  });
+  input.value = '';
+  renderTodos();
+}
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/195449725-faf07458-b8e0-4c93-a0db-68eb4ad99044.png)

- [ ] When you will add a new todo then a todo is an object with three key-value pairs which are **description**, **completed** and **id**. But when i added a todo in your application it missed the id in your key-value pairs as you can see in the image. Kindly fix it :+1:

Reply:
@HAMMAS-SALEEM ok I will fix that tonight

Reply:
@HAMMAS-SALEEM  it should be ok now

Reply:
ok it is working now